subsys/bluetooth/controller/ll_sw/lll_filter.h

```diff
@@ -13,12 +13,43 @@ struct lll_filter {
 	u8_t  bdaddr[WL_SIZE][BDADDR_SIZE];
 };
 
+/* Whitelist peer list */
+struct white_list {
```

Comment:
I think the structures should have `lll_filter` as prefix to follow general naming conventions.

Reply:
Good point - however - shouldn't it be for instance lll_whitelist? To be in line with struct lll_filter {}

Reply:
There is a bit of mess here, naming wise - ULL vs. LLL I mean? In a way they should be named ull_whitelist/resolvelist, as they are 'at home' in ull_filter, on the other hand lll_filter.h was used to hold the ull_....() API functions and lll_filter is still called lll_filter.